[Bug]: delete-banner script fails with MODULE_NOT_FOUND due to incorrect model import

### What happened?

The config/delete-banner.js script fails to execute due to an error locating the Banner model. This occurs because the script uses module-alias with a base path that, combined with the ~/models/schema/banner import, incorrectly attempts to resolve the model path to /app/api/models/schema/banner. The update-banner.js script, which functions correctly, uses @librechat/data-schemas to import models.

### Version Information

ghcr.io/danny-avila/librechat-dev-api            latest          6018fc705f40   8 hours ago     1.11GB
ghcr.io/danny-avila/librechat-rag-api-dev-lite   latest          2d989d133714   9 months ago    1.86GB

### Steps to Reproduce

1. Run the delete banner script docker exec -it LibreChat-API /bin/sh -c "cd .. && npm run delete-banner"

### What browsers are you seeing the problem on?

Microsoft Edge

### Relevant log output

```shell
$ docker exec -it LibreChat-API /bin/sh -c "cd .. && npm run delete-banner"

> LibreChat@v0.7.8 delete-banner
> node config/delete-banner.js

node:internal/modules/cjs/loader:1215
  throw err;
  ^

Error: Cannot find module '/app/api/models/schema/banner'
Require stack:
- /app/config/delete-banner.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1212:15)
    at Module._resolveFilename (/app/node_modules/module-alias/index.js:49:29)
    at Module._load (node:internal/modules/cjs/loader:1043:27)
    at Module.require (node:internal/modules/cjs/loader:1298:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/app/config/delete-banner.js:4:16)
    at Module._compile (node:internal/modules/cjs/loader:1529:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)
    at Module.load (node:internal/modules/cjs/loader:1275:32)
    at Module._load (node:internal/modules/cjs/loader:1096:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/app/config/delete-banner.js' ]
}

Node.js v20.19.2
```

### Screenshots

_No response_

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/config/delete-banner.js
+++ b/config/delete-banner.js
@@ -1,7 +1,8 @@
 const path = require('path');
+const mongoose = require(path.resolve(__dirname, '..', 'api', 'node_modules', 'mongoose'));
+const { Banner } = require('@librechat/data-schemas').createModels(mongoose);
 require('module-alias')({ base: path.resolve(__dirname, '..', 'api') });
 const { askQuestion, silentExit } = require('./helpers');
-const Banner = require('~/models/schema/banner');
 const connect = require('./connect');
 
 (async () => {

--- a/config/reset-password.js
+++ b/config/reset-password.js
@@ -1,8 +1,9 @@
 const path = require('path');
 const bcrypt = require('bcryptjs');
 const readline = require('readline');
+const mongoose = require(path.resolve(__dirname, '..', 'api', 'node_modules', 'mongoose'));
+const { User } = require('@librechat/data-schemas').createModels(mongoose);
 require('module-alias')({ base: path.resolve(__dirname, '..', 'api') });
-const User = require('../api/models/User');
 const connect = require('./connect');
 
 const rl = readline.createInterface({

